hcp-ui-templates/ec2-existing-vpc/main.tf

```diff
@@ -94,7 +115,34 @@ output "nomad_url" {
 output "hashicups_url" {
   value = "http://${module.aws_ec2_consul_client.public_ip}"
 }
-
+output "consul_export" {
+  value     = <<EOF
+  export CONSUL_HTTP_ADDR="${hcp_consul_cluster.main.consul_public_endpoint_url}"
+  export CONSUL_HTTP_TOKEN="${hcp_consul_cluster_root_token.token.secret_id}"
```

Comment:
I think it'd be easier to export each of these separately for cleanliness. So like:

```terraform
output "consul_http_addr" {
...
}

output "consul_http_token" {
...
}
```

The `export $VAR=` is a shell-specific command that isn't really relevant to terraform, and we can combine that with the `export` bit in your `how_to_connect` output below